src/common.h

```diff
@@ -125,7 +125,10 @@ int ishexn(const char *q, int n);
 int ishexucn(const char *q, int n);
 int ishexlcn(const char *q, int n);
 /* length of hex. if extra_chars not null, it will be 1 if there are more
- * non-hex characters after the length of valid hex chars returned. */
+ * non-hex characters after the length of valid hex chars returned.
+ * NOTE, the return will always be an even number (rounded down). so if we
+ * want the length of "ABCDE", it will be 4 not 5.
+ */
```

Comment:
Does "ABCDE" not trigger "extra_chars"? Given it returns 4, it should.

Reply:
I am not sure.  Let me check.....

yes, hexlen("ABCDE", &ex)    returns 4, and ex is set to 1.

Reply:
Good, then we're fine!